app/assets/javascripts/modules/hide-other-links.js

```diff
@@ -79,8 +79,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     event.preventDefault()
 
     this.hiddenElementContainer.style.display = ''
+    this.hiddenElementContainer.querySelectorAll('a')[0].focus()
```

Comment:
Do you mean using index [0] to target the first anchor is brittle? I'm unclear of the benefit of adding a `js-` class to the button to fix this. Perhaps a better solution is to target the first anchor by class name, as they all have a `govuk-link` class. The reason I was avoiding that in case of the possibility that these anchors might not have valid classes, but checking their origin the class is assigned in the presenter so that seems safe to use.

Reply:
Oh I got confused. Ignore my original point - the solution I suggested wasn't valid.